ui/cypress/support.js

```diff
@@ -1,23 +1,21 @@
-/* global Cypress, cy */
+/* global Cypress, cy, beforeEach */
 import '@testing-library/cypress/add-commands';
 import installLogsCollector from 'cypress-terminal-report/src/installLogsCollector.js';
 
 installLogsCollector();
 
-Cypress.Commands.add('login', (username = 'idtest0', password = '0000') => {
+beforeEach(() => {
   cy.visit('/');
+});
+
+Cypress.Commands.add('login', (username = 'idtest0', password = '0000') => {
   cy.findByRole('heading', { name: 'MXCuBE' }).should('be.visible');
   cy.findByLabelText('Login ID').type(username);
   cy.findByLabelText('Password').type(password);
   cy.findByRole('button', { name: 'Sign in' }).click();
 });
 
-Cypress.Commands.add('takeControl', (returnPage = '/datacollection') => {
-  /* firefox (only firefox) throws an unhandled promise error when executing
-     this function. Hence, we tell cypress to ignore this, otherwise the tests
-     fail, when we try to click the observer mode dialog away. */
-  Cypress.on('uncaught:exception', () => false);
```

Comment:
I'm not seeing those unhandled promise errors any more. :tada:

Reply:
Great! This is probably another benefit from updating cypress and firefox recently